### PR TITLE
Day 2

### PR DESCRIPTION
**Pull request by khola**

### CODE REVIEW

File: 02/run.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    return matches.count(True) == 1
+
+
+positives_part_1 = 0
+positives_part_2 = 0
+
+for password in passwords:
+    parts = password.split(" ")
+    limits = list(map(int, parts[0].split("-")))
+    letter = parts[1].split(":")[0]
```
Comment (janosio) [inline 02/run.py]: `letter = parts[1][0]` is enought

File: 02/run.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    word = parts[2]
+    if check_password_1(limits[0], limits[1], letter, word):
+        positives_part_1 = positives_part_1 + 1
```
Comment (janosio) [inline 02/run.py]: `positives_part_1 += 1`

File: 02/run.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
+with open("input", "r") as file:
+    passwords = file.readlines()
+
+
+def check_password_1(min, max, letter, word):
+    count = list(word).count(letter)
+    return bool(count >= min and count <= max)
```
Comment (wojtii) [inline 02/run.py]: No need for bool here:
```suggestion
    return count >= min and count <= max
```
Or:
```suggestion
    return min <= count <= max
```

**Review by wojtii — COMMENTED** [review]
Looks good 👍

**Pull request merged**